lib/debug.ml

```diff
@@ -112,14 +73,13 @@ let gettimestring () =
     (int_of_float (1000.0 *. msec))
 
 let format include_time brand priority message =
-  let host = Cache.get hostname in
   let id = get_thread_id () in
   let name = match ThreadLocalTable.find names with Some x -> x | None -> "" in
   let task = match ThreadLocalTable.find tasks with Some x -> x | None -> "" in
 
-  Printf.sprintf "[%s%5s|%s|%d %s|%s|%s] %s"
+  Printf.sprintf "[%s%5s|%d %s|%s|%s] %s"
```

Comment:
This changes the log format, it might break various tools that parse logs, such as RAGE importers, CIS health check, etc. Can we keep the pipe and just have an empty value there? Better than subtly breaking log parsing.

Reply:
Are you sure it will break things? Cause it's kinda ugly to just leave an empty pipe there.
But if it will break things I'll do that.

Reply:
Ok I've updated it to preserve the old format for now